src/components/screen/__tests__/Chat.test.tsx

```diff
@@ -67,7 +48,36 @@ describe('[Chat] interaction', () => {
     fireEvent.press(chatBtn);
   });
 
-  afterAll(() => {
-    cleanup();
+  describe('dispatch showModal', () => {
+    it('should dispatch [show-modal] when peerImage is pressed', () => {
+      jest
+        .spyOn(ProfileContext, 'useProfileContext')
+        .mockImplementation(() => ({
+          showModal: jest.fn(),
+          state: null,
+        }));
+      const chatListItem = testingLib.queryByTestId('CHAT_LIST_ITEM0');
+      act(() => {
+        fireEvent.press(chatListItem);
+      });
+    });
+
+    it('should call [show-modal] when modal is available', () => {
+      jest
+        .spyOn(ProfileContext, 'useProfileContext')
+        .mockImplementation(() => ({
+          showModal: jest.fn(),
+          state: {
+            user: null,
+            deleteMode: true,
+            modal: jest.mock,
+          },
+        }));
+      const chatListItem = testingLib.queryByTestId('CHAT_LIST_ITEM0');
+      testingLib.rerender(component);
+      act(() => {
+        fireEvent.press(chatListItem);
+      });
```

Comment:
It's better to expect results after firing the event!

Reply:
I added test code! thx!